spec/models/weight_limits_spec.rb

```diff
@@ -91,6 +101,171 @@ def build_inventory_unit(variant, order)
       Spree::ActiveShipping::Config.set(:default_weight => 1)
     end
 
+    describe ".valid_weight_for_package?" do
```

Comment:
Usually we do not test private methods, they should be tested as part of the behavior of the public method that they are called from. In this particular case the `valid_weight_for_package?` helper method should be tested as part of the expected behavior for the `ActiveShipping::Base#available?`. The tests for that should probably live in a `spec/models/spree/calculator/shipping/active_shipping/base_spec.rb` file however. Currently there are some tests for that in `spec/models/active_shipping_calculator_spec.rb` but we should probably move those in the correct place.